### ISSUE TITLE
cy.reload() should return the window object of the newly reloaded page

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#105]
cy.reload() should return the window object of the newly reloaded page, but it is currently undefined.

**Example Code**
```javascript
      cy.reload().then(function(win){
        debugger
      })
```

![screen shot 2016-02-11 at 1 09 20 pm](https://cloud.githubusercontent.com/assets/1271364/12985461/c39791b8-d0c0-11e5-825f-31666dd2a544.png)

**Comment by jennifer-mann** [comment]
This is fixed in the upcoming `0.14.0` release.

**Comment by brian-mann** [comment]
Fixed in [`0.14.0`](https://github.com/cypress-io/cypress/wiki/Changelog#0140-03082016).
